lib/modules/manager/nuget/update.ts

```diff
@@ -1,8 +1,9 @@
 import semver, { ReleaseType } from 'semver';
-import { XmlDocument } from 'xmldoc';
+import { XmlDocument, XmlElement } from 'xmldoc';
```

Comment:
```suggestion
import { XmlDocument, type XmlElement } from 'xmldoc';
```

Reply:
The vim syntax highlighting is unhappy about that. Looks like it's supposed to support it. Another bug to file.